charts/sample_local_helm_setup.sh

```
@@ -17,12 +17,20 @@ echo "Using local chart: $USE_LOCAL_CHART"
 echo "Setting up MCP Gateway using Helm chart..."
 
 # Create Kind cluster with inline configuration (skip if already exists)
+<<<<<<< Updated upstream
```

Shellcheck (0.11.0)

[error] Couldn't parse this here string. Fix to allow more checks.

(SC1073)